iosApp/iosApp/Pages/AlertDetails/AlertDetails.swift

```diff
@@ -268,12 +279,17 @@ struct AlertDetails: View {
         let scrollContent = ScrollView {
             VStack(alignment: .leading, spacing: 24) {
                 alertTitle
-                alertPeriod
-                VStack(alignment: .leading, spacing: 16) {
-                    affectedStopCollapsible
-                    tripPlannerLink
+                if !isElevatorClosure {
+                    alertPeriod
+                    VStack(alignment: .leading, spacing: 16) {
+                        affectedStopCollapsible
+                        tripPlannerLink
+                    }
                 }
                 alertDescription
+                if isElevatorClosure {
+                    alertPeriod
```

Comment:
The alert period lis a bit squished. I see this also is an issue for other alerts. If minor, could you include the fix here?
![IMG_0222](https://github.com/user-attachments/assets/5ed98219-b670-4050-bbea-7f9e79b5bfdf)

Reply:
I did wonder when I implemented this on Android why the structure was the way that it was on iOS. This seems better:
<img width="446" alt="Screenshot 2025-01-31 at 2 43 40 PM" src="https://github.com/user-attachments/assets/c9e39eec-ca93-4377-b187-45e6a0670343" />